handlers/leagues.js

```diff
@@ -43,7 +45,7 @@ var onLeaguesInMonthResponse = function onLeaguesInMonthResponse(message, callba
 
     if (response.eresult === 1) {
         if (this.debug) util.log("Received leagues in month response " + response.eresult);
-        this.emit("leaguesInMonthData", response.eresult, response);
+        this.emit("leaguesInMonthData", response.month, response.year, response.leagues);
```

Comment:
Yeah, I guess you are right. This particular method though is unique in the way that the eresult has a default of 2, which is fail as I recall. This implies that it might not always be set.